opensrp-reveal/src/main/java/org/smartregister/reveal/repository/RevealRepository.java

```diff
@@ -81,6 +83,8 @@ public void onCreate(SQLiteDatabase database) {
         TaskRepository.createTable(database);
         LocationRepository.createTable(database);
         StructureRepository.createTable(database);
+        ClientFormRepository.createTable(database);
```

Comment:
Add migration scripts for these two DBs if you are going to release to running prod

Reply:
done